app/Models/User.php

```diff
@@ -105,4 +105,18 @@ public function getDisplayName()
         return $display_name;
     }
 
+    public function printUserType()
+    {
+        $user_type = $this->user_type;
+
+        if ($user_type == 1) {
+            echo "Admin";
+        }else if ($user_type == 2) {
+            echo "Author";
+        }else{
+            echo "Regular User";
+        }
```

Comment:
Please use `return "value";` instead of direct `echo`. Blade's `{{ ... }}` do the echoing.
Please correct also the spacing of the code.